CONTRIBUTING.md

```diff
@@ -57,50 +280,95 @@ add an entry to the NCL Index in cases where a full **NCL Entry** is not
 than waiting for a full **NCL Entry** to be completed.
 
-## Adding Content
+1. Create a new file in `ncl/ncl_receipts/` based off of the
+   `templates/receipt_template.ipynb` template.
 
-### Python Entry
+1. See below for adding the covered functions to the NCL Index
 
-1. If the content is primarily visualization, create a new file in the
-   appropriate directory in `applications/` based off of
-   the `templates/viz_template.ipynb`
+1. Make sure to clear and run all outputs before asking for a review
 
-1. If the content is primarily computational (even if it includes
-   visualization), create a new file in the appropriate directory
-   in `applications/` based off of the `templates/computational_template.ipynb`
+1. Add a new line to the `ncl/ncl_index/ncl-index-table.csv` file
 
-1. If relevant, link to corresponding NCL content at the bottom of the file
+### Generate the Documentation Locally
+At the base of the reposistory `geocat-applications/`, run
+```
+make clean html
+```
+This will build the documentation locally.
 
-1. [INTEND TO AUTOMATE IN THE FUTURE] Add the new file to the `.rst` file in the
-   same directory in `applications/` as your new file to add it to the webpage's
-   TOC.
+New documentation can be viewed by opening `index.html` generated under
+ `_build/html/index.html` on a local browser
 
-1. [INTEND TO AUTOMATE IN THE FUTURE] Add the new file to
-   the `applications/applications.rst` file to add it to the cards on the main
-   page of the website.
+```
+firefox _build/html/index.html
+```
 
-1. Make sure to clear and run all outputs before asking for a review
+## Contribute the Code
+Once you have completed your changes on notebooks and they are ready for review
+by the GeoCAT team, you can open a new Pull Request. This section describes how
+to open a pull request and what to expect after you open it.
 
-### NCL Entry
+### Check Files Changed
 
-1. Create a new file in `ncl/ncl_entries/` based off of the
-   `templates/ncl_template.ipynb` template.
+For a Python Entry, the files changed should include:
+- New Python page under `applications`
+- Link to directory in `.rst` file in the same directory as `applications/`
+- Link new file in `applications/applications.rst` file
 
-1. Add the file to the `ncl/ncl_entries/ncl_entries.rst` file
+For a NCL to Python entry, the files changes should include:
+- A new receipts entry under `receipts`
+- A NCL to Python notebook under `ncl_entries`
+- A new row in `ncl-index-table.csv`
 
-1. See below for adding the covered functions to the NCL Index
+### Open a new Pull Request
+A Pull Request is a request to merge code from your local fork to the main repository.
+GitHub has extensive [pull request guides and documentation](https://docs.github.com/en/pull-requests)
+if you'd like more information
 
-1. Make sure to clear and run all outputs before asking for a review
+On your local fork of the `geocat-applications` repo:
```

Comment:
Should this be something more like "on your fork of `geocat-applications` on GitHub"?

Specifically, the terminology "local fork" is a little confusing to me.  Are you describing how to open a PR in VS Code?  I don't know that we want to presume everyone will be using the same IDE.

I think just copying over the section from GeoCAT-comp could work fine here for this full section probably.